samples/azure/storage-queues/ASQ_9/Shared/BackwardsCompatibleQueueNameSanitizer.cs

```diff
@@ -0,0 +1,91 @@
+﻿using System;
+using System.Security.Cryptography;
+using System.Text;
+using System.Text.RegularExpressions;
+
+#region BackwardsCompatibleQueueNameSanitizer
+
+public static class BackwardsCompatibleQueueNameSanitizer
```

Comment:
We no longer support this right?

Reply:
No, we don't. It's left in to show how to get the backward compatible sanitization _if_ customers need to.